public/app/features/explore/LiveLogs.tsx

```diff
@@ -23,17 +24,15 @@ const getStyles = (theme: GrafanaTheme) => ({
   logsRowFresh: css`
     label: logs-row-fresh;
     color: ${theme.colors.text};
-    background-color: ${selectThemeVariant(
-      { light: theme.background.logsFresh, dark: theme.background.logsFresh },
-      theme.type
-    )};
+    background-color: ${tinycolor(theme.colors.blueLight)
+      .setAlpha(0.25)
+      .toString()};
```

Comment:
@aocenas @dprokop I used tinycolor library that is already in Grafana. I think this is most readable/understandable solution. Thank you both for tips and advices!